src/lib/kclCommands.ts

```diff
@@ -149,9 +164,26 @@ export function kclCommands(
             }
             return value
           },
-          options: providedOptions,
+          options: commandProps.specialPropsForSampleCommand.providedOptions,
         },
       },
     },
+    {
+      name: 'share-file-link',
+      displayName: 'Share file',
+      description: 'Create a link that contains a copy of the current file.',
+      groupId: 'code',
+      needsReview: false,
+      icon: 'link',
+      hide: 'both',
```

Comment:
@franknoirot After your last commit the file menu button is disabled, but I was still able to Cmd+K the command. Is that expected?

Reply:
Hm it is not, I expected `hide: 'both'` to remove it. Apologies, let me fix that real quick.

Reply:
I just commented it out lol whatever